lib/functions.php

```diff
@@ -80,6 +77,9 @@ function html_email_handler_send_email(array $options = null) {
 		$options["bcc"] = array($options["bcc"]);
 	}
 	
+	$options['html_message'] = html_email_handler_make_html_body($options);
+	$options['plaintext_message'] = $options['body'];
```

Comment:
i think they should only be filled when not empty.

no need to check the plugin setting, this function is there to make sure you can send html mail regardless of the setting. Only the hooks need to respect the plugin setting